api/rest/restapi.go

```diff
@@ -585,6 +585,69 @@ func (api *API) allocationHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// defined groups / aliases for filtering
+var filterGroups = map[string][]string{
+	"error":  {"error", "pin_error", "unpin_error", "cluster_error"},
+	"queued": {"queued", "pin_queued", "unpin_queued"},
+}
+
+// check if any of the strings in matches are in the list
+func anyStringInSlice(matches, list []string) bool {
```

Comment:
Is using a very short definition like 'find' an acceptable practice here? Should this function be moved into a general utilities file instead of restapi.go ? Your stolen parameter names, though, are spot on.